cluster-operator/src/test/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2ClusterTest.java

```diff
@@ -413,6 +413,53 @@ public void testGenerateDeploymentWithScramSha512Auth() {
         assertThat(AbstractModel.containerEnvVars(cont).get(KafkaMirrorMaker2Cluster.ENV_VAR_KAFKA_CONNECT_SASL_MECHANISM), is("scram-sha-512"));
     }
 
+    @Test
+    public void testGenerateDeploymentWithScramSha512AuthAndTLSSameSecret() {
```

Comment:
I do wonder if a small JavaDoc comment expanding on why the test of the same secret is important and how without the deduplication of the volumes it would result in a Kube error

Reply:
Javadoc added